docs/SetupAndTeardown.md

```diff
@@ -169,7 +173,13 @@ describe('describe outer', () => {
 // test 3
 ```
 
-Just like the `describe` and `test` blocks Jest calls the `before*` and `after*` hooks in the order of declaration. Note that the `after*` hooks of the enclosing scope are called first. For example, here is how you can set up and tear down resources which depend on each other:
+Just like the `describe` and `test` blocks Jest calls the `before*` and `after*` hooks in the order of declaration.
+
+:::note
+
+The `after*` hooks of the enclosing scope are called first. For example, here is how you can set up and tear down resources which depend on each other:
+
+:::
```

Comment:
Also here there is no need to change anything. It does not look good or logic to end the note with a colon. Also this splitting does not make sense, because this is just one piece of text.